association.go

```diff
@@ -2022,13 +2022,23 @@ func (a *Association) resetStreamsIfAny(p *paramOutgoingResetRequest) *packet {
 			a.name, p.senderLastTSN, a.peerLastTSN)
 		result = reconfigResultInProgress
 	}
-
-	return a.createPacket([]chunk{&chunkReconfig{
-		paramA: &paramReconfigResponse{
-			reconfigResponseSequenceNumber: p.reconfigRequestSequenceNumber,
-			result:                         result,
+	rsn := a.generateNextRSN()
+	tsn := a.myNextTSN - 1
+	return a.createPacket([]chunk{
+		&chunkReconfig{
+			paramA: &paramReconfigResponse{
+				reconfigResponseSequenceNumber: p.reconfigRequestSequenceNumber,
+				result:                         result,
+			},
 		},
-	}})
+		&chunkReconfig{
+			paramA: &paramOutgoingResetRequest{
+				reconfigRequestSequenceNumber:  rsn,
+				reconfigResponseSequenceNumber: p.reconfigRequestSequenceNumber,
+				senderLastTSN:                  tsn,
+				streamIdentifiers:              p.streamIdentifiers,
+			},
+		}})
```

Comment:
This chunk is a request and may drop - need a retransmission.
Are you able to add a couple of more lines here to do it? You could refer to [this section of the code](https://github.com/pion/sctp/blob/d0b7cf3963e99337bdf09e11989cf8b0916e1179/association.go#L684-L708) and do the same.

Reply:
OK, will update it today.

Reply:
@enobufs updated, hope it correct.